### DISCUSSION TITLE
Error building with PXWeb as input

### DISCUSSION THREAD

**Discussion by karihj** [open-sdg/open-sdg#1968] [Q&A]
At Statistics Faroe Islands we currently are publishing both in CSV-format for the Open SDG-site (https://sdg.hagstova.fo/sdg-site/en/) and in our PXWeb Database (https://statbank.hagstova.fo/pxweb/en/H2/H2__SDG/). 
However, we are now exploring the option of switching to PXWeb as the only source, as this would streamline our publishing process and make it more efficient.
 
To facilitate this transition, we have built a new data and site repository that you can access via the following links:
 
•	https://github.com/hagstovan/px-sdg-data/
•	https://github.com/hagstovan/px-sdg-site
•	https://hagstovan.github.io/px-sdg-site/1-2-1/ (Site) 
 
We have used the same setup as Statistics Palestine Demo setup (https://github.com/brockfanning/palestine-open-sdg-pxweb-demo/blob/develop/config_data.yml#L48-L226), albeit with only one indicator to test it out. 

Our configuration file can be found here:
•	https://github.com/hagstovan/px-sdg-data/blob/develop/config_data.yml (from line 42)
 
Unfortunately, when we try to build, we encounter an error that I am unable to debug. We have set the "Year" variable to 'year' as it is called in the returned PX-file. Since there is no dedicated "Value" column, I have used 'value', as was done in the Palestinian version.
 
Would it be possible for you to assist us in troubleshooting this issue?

**Comment by christoph-LSN** [comment]
@karihj I am not sure, but have you tried "Value" instead of "value"

I had the following Error using "value" instead of "Value" in my CVS file:

ValueError: 'Value' is not in list
Error: Process completed with exit code 1.

https://github.com/christoph-LSN/IM_Data/actions/runs/4629151547/jobs/8189064777

**Reply by karihj** [reply]
Thank you for the suggestion - I have tried it with 'Value' instead of 'value' however with the same result

**Comment by sarah-beasley** [comment]
Just to update on this, we are working on a fix for this to go into the next release of Open SDG (2.3.0). Will keep you posted!

**Reply by karihj** [reply]
Thank you Sarah for the update! If we can assist you with anything, please feel free to contact us
